src/main/java/com/example/demo/service/UserService.java

```diff
@@ -121,11 +125,23 @@ private GameDTO convertToDTO(Game game) {
     }
 
     public List<GameDTO> getList(Long userId) {
-        User user = userRepository.findById(userId)
-                .orElseThrow(() -> new IllegalArgumentException("Not found User with id = " + userId));
 
-        return user.getGames().stream()
-                .map(this::convertToDTO)
-                .collect(Collectors.toList());
+        Optional<User> userOptional = userRepository.findById(userId);
+        if (userOptional.isPresent()) {
+            User user = userOptional.get();
+            return user.getGames().stream()
+                    .map(this::convertToDTO)
+                    .collect(Collectors.toList());
+        } else {
+            throw new IllegalArgumentException("User not found with id: " + userId);
+
+        }
+
+    }
+
+    public Long getUsernameInToken(String token) {
+        String username = jwtService.extractUsername(token.substring(7));
```

Comment:
why do you use this hardcoded number 7 ?

Reply:
to delete the bearer in the token